src/styles/atoms/_atoms.input.scss

```diff
@@ -322,12 +331,20 @@ textarea {
     --input-height: var(--spacer-l);
     --input-icon-size: 1rem;
     --input-padding-horizontal: var(--spacer-xs);
+    --select-padding-vertical: 0;
     --textarea-height: 8rem;
+
+    select {
+      @-moz-document url-prefix() { // sass-lint:disable-line no-vendor-prefixes
+        line-height: 2.1; // Fixes Firefox line-height bug
```

Comment:
Indien mogelijk nog een link naar StackOverflow (of iets dergelijks) toevoegen.

Reply:
In v5 van de core branding kwam die bug niet voor in Firefox. Ik had in v6 bovenstaand stukje code verwijderd, omdat ik dacht dat die fix niet meer relevant was. Blijkbaar dus nog wel. 😅 

Dus ik heb gewoon de fix uit v5 terug toegepast, maar daar stond dus ook niet echt een link of iets dergelijks bij. Ik zal een comment/link toevoegen. 🙂